packages/google_maps_flutter/google_maps_flutter_web/lib/src/convert.dart

```diff
@@ -403,13 +403,51 @@ gmaps.PolylineOptions _polylineOptionsFromPolyline(
   final List<gmaps.LatLng> paths =
       polyline.points.map(_latLngToGmLatLng).toList();
 
+  final bool hasPattern = polyline.patterns.isNotEmpty;
+
+  final List<gmaps.IconSequence> iconSequences = <gmaps.IconSequence>[];
+
+  if (hasPattern) {
+    List<Object>? getPattern(List<List<Object>> input, String patternName) {
+      final List<Object> found = input.firstWhere(
```

Comment:
This might be good for a first version, where we want to have dashed lines, but isn't this algorithm potentially discarding more complex PatternItem combinations? For example: 

`dot, gap, dot, gap, line, gap`

Wouldn't work, right?